src/pages/Home.js

```diff
@@ -866,15 +993,31 @@ const Home = props => {
         style={{ marginTop: "-10px", height: "400px" }}
       >
         <Grid item xs={12}>
-          <iframe
+          <GoogleMapReact
+            center={defaultPropsMaps.center}
+            zoom={defaultPropsMaps.zoom}
+            options={{
+              styles: props.theme.mode === "dark" && mapStyle
+            }}
+            bootstrapURLKeys={{
+              key: "AIzaSyB3JAqF0AMxgbfnGtt0R5Yk0MaBe9p1P5s"
+            }}
+          >
+            <AnyReactComponent
+              lat={ThapadUniversity[0]}
+              lng={ThapadUniversity[1]}
+              text="Thapad University"
```

Comment:
Can we please change the `text` attribute from "Thapad" to "Thapar"

Reply:
Sure thing, I missed it.